pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -36,6 +37,18 @@ func TestValidateKubeSchedulerConfiguration(t *testing.T) {
 		HealthzBindAddress:             "0.0.0.0:10254",
 		MetricsBindAddress:             "0.0.0.0:10254",
 		HardPodAffinitySymmetricWeight: 80,
+		TopologySpreadConstraints: []corev1.TopologySpreadConstraint{
```

Comment:
Can we add more tests to cover most of the validation logic? e.g. expect error when nodeSelector is not nil, etc.

Reply:
There is that case lines below.